201-blockchain-asaservice/README.md

```diff
@@ -26,13 +23,12 @@ This template deploys a **Azure Blockchain Service**. Azure Blockchain Service i
 
 If you are new to BlockChain Services, see:
 
-- [Azure BlockChain Services](https://azure.microsoft.com/services/blockchain-service/).
```

Comment:
The marketing name is Azure Blockchain Service

Reply:
Thanks Pat, I fixed it.